cmd/cli/kubectl-kyverno/processor/policy_processor.go

```diff
@@ -214,6 +217,35 @@ func (p *PolicyProcessor) ApplyPoliciesOnResource() ([]engineapi.EngineResponse,
 		responses = append(responses, validateResponse)
 		resource = validateResponse.PatchedResource
 	}
+	// Mutate Admission Policies
+
+	mapResponses := make([]engineapi.EngineResponse, 0, len(p.MutatingAdmissionPolicies))
+	for _, mapPolicy := range p.MutatingAdmissionPolicies {
+		policyData := admissionpolicy.NewMutatingPolicyData(mapPolicy)
+
+		for _, binding := range p.MutatingAdmissionPolicyBindings {
+			if binding.Spec.PolicyName == mapPolicy.Name {
+				policyData.AddBinding(binding)
+			}
+		}
+
+		mutateResponse, err := admissionpolicy.Mutate(policyData, resource)
+		if err != nil {
+			log.Log.Error(err, "failed to apply MAP", "policy", mapPolicy.Name)
+			continue
+		}
+
+		if !mutateResponse.IsEmpty() {
+			mapResponses = append(mapResponses, mutateResponse)
+			p.Rc.AddMutatingAdmissionPolicyResponse(mutateResponse)
```

Comment:
You should remove this, as the function [addMutateResponse()](https://github.com/kyverno/kyverno/blob/ada597abf4915e3f45277de36ce48dfd48e2e808/cmd/cli/kubectl-kyverno/processor/result.go#L86-L120) already does the same thing. But, you should modify the function so that it processes the MAPs response.